astropy/modeling/functional_models.py

```diff
@@ -545,6 +550,43 @@ def fit_deriv(x, *params):
         return [d_factor]
 
 
+class Multiply(Fittable1DModel):
```

Comment:
Thanks for agreeing to write this model. It will save us additional work with internal code.
I think the name is too general although I'm not sure what would be a good name.
Another suggestion is to name it similarly to `RedshiftScaleFactor` - `QuantityScaleFactor`?

In any case it needs to be added to `__all__`.

Reply:
I think `Multiply` does a reasonably good job of describing what it does, given that evaluate is `factor * x` :wink:.

It should also pretty much work in most other ways the `*` operator works as long as you have numeric inputs.

Reply:
OK, fine.

You also need to add a change log entry.
I'm not sure why CircleCi is having issues but once it's resolved this can be merged.
Thanks for the work.

Reply:
Naming things is hard :laughing:

Reply:
Pshaw, naming things is easy. Giving *good* names is hard 😉